scripts/source-maintenance.sh

```diff
@@ -117,7 +145,7 @@ if test "${ASVER}" != "${TargetAstyleVersion}" ; then
 		echo "Formatting anyway, please double check output before submitting"
 	fi
 else
-	echo "Found astyle ${ASVER}. Formatting..."
+	echo "Found astyle ${ASVER}"
 fi
 
```

Comment:
This is where export should be used (instead of above)
```suggestion
export ASTYLE

```

Reply:
Now that scripts/formater.pl no longer receives ASTYLE environment variable the export can simply be removed.